lib/src/native/realm_core.dart

```diff
@@ -272,34 +276,48 @@ class _RealmCore {
 
   RealmObjectHandle createRealmObjectWithPrimaryKey(Realm realm, int classKey, Object primaryKey) {
     return using((Arena arena) {
-      final realm_value = _toRealmValue(primaryKey, arena);
-      final realmPtr = _realmLib.invokeGetPointer(() => _realmLib.realm_object_create_with_primary_key(realm.handle._pointer, classKey, realm_value.ref));
+      final realmValue = _toRealmValue(primaryKey, arena);
+      final realmPtr = _realmLib.invokeGetPointer(() => _realmLib.realm_object_create_with_primary_key(realm.handle._pointer, classKey, realmValue.ref));
       return RealmObjectHandle._(realmPtr);
     });
   }
 
   Object? getProperty(RealmObject object, int propertyKey) {
     return using((Arena arena) {
-      final realm_value = arena<realm_value_t>();
-      _realmLib.invokeGetBool(() => _realmLib.realm_get_value(object.handle._pointer, propertyKey, realm_value));
-      return realm_value.toDartValue(object.realm!);
+      final realmValue = arena<realm_value_t>();
+      _realmLib.invokeGetBool(() => _realmLib.realm_get_value(object.handle._pointer, propertyKey, realmValue));
+      return realmValue.toDartValue(object.realm!);
     });
   }
 
   void setProperty(RealmObject object, int propertyKey, Object? value, bool isDefault) {
     return using((Arena arena) {
-      final realm_value = _toRealmValue(value, arena);
-      _realmLib.invokeGetBool(() => _realmLib.realm_set_value(object.handle._pointer, propertyKey, realm_value.ref, isDefault));
+      final realmValue = _toRealmValue(value, arena);
+      _realmLib.invokeGetBool(() => _realmLib.realm_set_value(object.handle._pointer, propertyKey, realmValue.ref, isDefault));
     });
   }
 
-  // ignore: unused_element
-  int get _threadId => _realmLib.get_thread_id();
+  int get threadId => _realmLib.get_thread_id();
```

Comment:
this one I kept to ease debugging. Why expose it publicly?

Reply:
I think I just fixed a warning. Will ignore instead.